packages/rest/src/lib/REST.ts

```diff
@@ -314,7 +315,14 @@ export class REST extends EventEmitter {
 	 * Runs a request from the api
 	 * @param options Request options
 	 */
-	public request(options: InternalRequest) {
+	public async request(options: InternalRequest) {
+		return parseResponse(await this.requestManager.queueRequest(options));
+	}
+
+	/**
+	 * Runs a request from the api, yielding the raw Response object
+	 */
```

Comment:
Missing the doc explaining what the parameter is:
```suggestion
	 * Runs a request from the api, yielding the raw Response object
	 * @param options Request options
	 */
```